src/test/java/seedu/address/logic/commands/VolunteerListEventCommandTest.java

```diff
@@ -0,0 +1,86 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalEvents.getTypicalEventStorage;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND;
+import static seedu.address.testutil.TypicalVolunteers.getTypicalVolunteerStorage;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.eventvolunteercommands.VolunteerListEventCommand;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.volunteer.Volunteer;
+
+public class VolunteerListEventCommandTest {
+    private Model model = new ModelManager(getTypicalEventStorage(), getTypicalVolunteerStorage(), new UserPrefs());
+
+    @Test
+    public void execute_invalidIndex_throwsCommandException() {
+        // Assign a volunteer to an event in the model
+        model.getEventStorage().getEventList().get(model.getFilteredEventList().size() - 1).addVolunteer(
+            model.getVolunteerStorage().getVolunteerList().get(model.getVolunteerStorage()
+                    .getVolunteerList().size() - 1));
```

Comment:
might wanna break this up into a few statements, its p confusing to read

Reply:
can split `model.getEventStorage().getEventList()` and `model.getVolunteerStorage().getVolunteerList()` into variables

Reply:
yes

Reply:
> yes

excellent feedback

Reply:
thx

Reply:
> > yes
> 
> excellent feedback

thx

Reply:
ok thanks for the suggestion